solr/src/main/resources/solr/conf/search-solr-managed-schema.xml

```diff
@@ -116,19 +116,25 @@
     <!-- docValues are enabled by default for long type so we don't need to index the version field  -->
     <field name="_version_" type="plong" indexed="false" stored="false"/>
 
-    <!-- CST fields -->
-    <field name="contents" type="text_general" indexed="true" stored="true" required="true"
-           multiValued="false" />
-    <field name="irm_uri_string" type="string" indexed="true" stored="true" required="true"
-           multiValued="false" />
-
-    <!-- If you don't use child/nested documents, then you should remove the next two fields:  -->
-    <!-- for nested documents (minimal; points to root document) -->
-    <field name="_root_" type="string" indexed="true" stored="false" docValues="false" />
-    <!-- for nested documents (relationship tracking) -->
-    <field name="_nest_path_" type="_nest_path_" /><fieldType name="_nest_path_" class="solr.NestPathField" />
-
-    <field name="_text_" type="text_general" indexed="true" stored="false" multiValued="true"/>
+  <!-- CST fields -->
+  <field name="contents" type="text_general" indexed="true" stored="true" required="true"
+    multiValued="false"/>
+  <field name="country_code" type="string" indexed="true" stored="true" required="false"
+    multiValued="false"/>
+  <field name="created" type="pdate" sortMissingLast="true"
+    indexed="true" stored="true" required="false" multiValued="false"/>
+  <field name="icid" type="string" indexed="true" stored="true" required="false"
+    multiValued="false"/>
+  <field name="irm_uri" type="string" indexed="true" stored="true" required="true"
```

Comment:
✏️ Should we be using `url` here instead? https://github.com/connexta/ion-index-api/pull/35#discussion_r347642762

Reply:
A fair question. I called it that because that was what is was called in SolrConfiguration. I think URL makes more sense. Here is the relevant snippet of DDF documentation (not sure that makes any difference to us, but here it is anyway)
![image](https://user-images.githubusercontent.com/133284/69257440-4c29a480-0b78-11ea-9647-fa6d8a186936.png)

Reply:
If even one person says "change it to say `resource_url`, I'd do it.

Reply:
This is not related to DDF. We should use URL from now on per Eric.